### DISCUSSION TITLE
[Request] Duplicati LXC

### DISCUSSION THREAD

**Discussion by dnldcs** [tteck/Proxmox#655] [Ideas]
[Duplicati](https://www.duplicati.com/) is  backup software to store encrypted backups online.

Duplicati is free software and open source. Source code is licensed under LGPL.

"Duplicati works with standard protocols like FTP, SSH, WebDAV as well as popular services like Backblaze B2, Tardigrade, Microsoft OneDrive, Amazon S3, Google Drive, box.com, Mega, hubiC and many others."

Seems like a more user friendly and gui-focused Rclone. 

I appreciate the consideration.

**Comment by devzwf** [comment]
dont want to be to much hard , but you should make some deep research on duplicati before starting using it :(
this software have a structural flow who have been keeping me far from it...
if you still want to use... please make sure you run a FULL DR test (including simulating loosing the db)
last update was in june :(

you should check kopia (https://kopia.io/) instead 

All what i write is my own opinion and should not be generalized .... just do your due diligence and research
